src/index.css

```diff
@@ -141,62 +209,69 @@ a:focus {
 
 .label {
   display: block;
-  font-size: 10px;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25rem;
   font-weight: bold;
-  color: #8f8888;
-  padding: 10px 0 4px 0;
-  margin: 0;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
As these three lines are getting repeated in different classes. The best practice is to create a new class and place these properties there instead of repeating the properties over and over again. It'll save your CSS lines as well 👍🏻 😄 
#### For Example
```css
.class-name {
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
}
```